packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/tools/Point.js

```diff
@@ -31,8 +33,25 @@ const Point = ({ active, coordinates, scale, tool }) => {
 
   if (!coordinates) return null
 
+  function select (e) {
+    console.log('select')
+    e.preventDefault()
+    e.stopPropagation()
+    setActive(true)
+  }
+
+  function deselect (e) {
+    console.log('deselect')
+    e.preventDefault()
+    e.stopPropagation()
+    setActive(false)
```

Comment:
Would active state be connected with whether or not the delete button shows? If so, then on pointer up, this mark should still be active.

Reply:
I copied `active` over from the original PR but I haven't 100% figured out what it means yet.

Reply:
Also noting that this event handler never runs. I'm not sure what's going on there.

Reply:
I've moved selecting marks into #1308.